docs/how-to/tls.rst

```diff
@@ -141,6 +141,13 @@ plaintext to TLS:
 
 3. run ``sudo snap restart microovn.daemon`` on **all** cluster members
 
+4. run ``sudo snap restart microovn.ovn-northd`` on **all** cluster members
+
+
+.. important::
```

Comment:
I suggest adding the admonishment before step 4. Something like:

Do not proceed to the next step until this current step is complete.

*How can the user ensure that step 3 is complete?*

Reply:
> How can the user ensure that step 3 is complete?

It should be fine as long as the services are not restarted at the same time. Honestly, even if they are restarted, I think the `northd` would self-recover, but as a precaution I think it's better if they are executed in sequence

Reply:
Maybe we could even drop the warning? Maybe I was being too dramatic :D

Reply:
Maybe just (in step 3):

"Allow the command to complete before proceeding to the next step."